### PR TITLE
UI Fixes and Revamped Developer List

### PR DESCRIPTION
**Pull request by Alphalaneous**
This PR fixes a ton of minor issues with the UI, including:

- Fixes misaligned Download counts due to non absolute button widths
- Fixes misaligned checkboxes due to content size changing from different sprites
- Replaces the exclamation point on the geode logo with the vanilla one
- Fixes the incorrect mod settings popup size that would get clipped on the top otherwise
- Fixes the oddly short mod problems popup
- Fixes small gap with border on mod settings

This PR also revamps the mod developer list to use a list rather than add a bunch of labels for each developer.

### CODE REVIEW

File: loader/src/ui/mods/list/ModItem.cpp [hunk 1]
```diff
@@ -379,7 +377,14 @@ void ModItem::updateState() {
             on->setColor(ccGRAY);
             on->setOpacity(105);
         }
+
+        // Make sure toggler is always the same size and position to prevent misalignment
+        m_enableToggle->setContentSize({35, 35});
```
Comment (HJfod) [inline loader/src/ui/mods/list/ModItem.cpp:382]: `CCMenuItemToggler::createWithStandardSprites` should just set the content size of the selected sprite to the same as the unselected sprite for a systematic fix of the root issue

**Review by HJfod — COMMENTED** [review]
i want to see the dev popup before approving

File: loader/src/ui/mods/list/ModItem.cpp (hunk 1 above)
Comment (Alphalaneous) [reply loader/src/ui/mods/list/ModItem.cpp:382]: That is what this uses, but it doesn't work, causing the checkbox to shift in the layout when the content size changes

**Comment by Alphalaneous** [comment]
Bindings fail moment

**Comment by snow111222-glitck2212222312** [comment]
it wont connect to the servers, it says "Couldn't resolve host name" can i have some help with this please

**Comment by Uproxide** [comment]
> it wont connect to the servers, it says "Couldn't resolve host name" can i have some help with this please

munch munch crunch crunches your skull

**Comment by Flobup** [comment]
what?

**Review by matcool — APPROVED** [review]

**Pull request merged**